api/controllers/SessionController.js

```diff
@@ -83,6 +83,14 @@ const finishOAuth = function (strategy, req, res, next) {
   return new Promise((resolve, reject) => {
     var respond = error => {
       if (error && error.stack) rollbar.handleError(error, req)
+      if (req.headers.accept === 'application/json') {
+        if (error) {
+          res.status(500).send(error)
```

Comment:
use res.serverError here -- it will skip the stack trace and also notify rollbar in production.